ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/base-fee-input/base-fee-input.js

```diff
@@ -142,6 +153,25 @@ const BaseFeeInput = () => {
     ],
   );
 
```

Comment:
Methods below can be extracted into small functional components and reused for base-fee and priority-fee